charts/px-backup/templates/pxcentral-post-install-hook.yaml

```diff
@@ -5,12 +5,6 @@ kind: Job
 metadata:
   name: pxcentral-post-install-hook
   namespace: {{ .Release.Namespace }}
-  annotations:
-    helm.sh/hook: post-install,post-upgrade
-    helm.sh/hook-weight: "5"
-    helm.sh/hook-delete-policy: before-hook-creation
-    meta.helm.sh/release-name: {{ .Release.Name }}
```

Comment:
we should have release name and namespace annotation.

Reply:
Added